### ISSUE TITLE
geogrid generated landmask does not see lakes with modis_15s_lake

### ISSUE THREAD

**Issue by markelg** [wrf-model/WPS#132]
There seems to be a "21" missing in GEOGRID.TBL in the landmask definition for modis_15s_lake

```
        landmask_water = modis_15s_lake:17         # Calculate a landmask from this field
        landmask_water =      modis_30s:17         # Calculate a landmask from this field
        landmask_water = modis_30s_lake:17,21      # Calculate a landmask from this field
```

This causes the LANDMASK computed by geogrid to not to see the lakes. I am not sure about the consequences, I think that real.exe fixes this afterwards recomputing the landmask, but at least this screws up my domain plotting script which uses geo_em files.

**Comment by mgduda** [comment]
I agree that we seem to be missing category 21 from the `landmask_water` specification for `modis_15s_lake`.

**Comment by mgduda** [comment]
@markelg Thanks very much for pointing out this issue. I've just merged PR #146 , which fixes this issue.
